src/frontend/utils/telemetry/Instrumentation.js

```diff
@@ -1,41 +0,0 @@
-// Copyright The OpenTelemetry Authors
```

Comment:
It looks like this was deleted because swo auto creates the SDK?

Reply:
Yes